linux/README.md

```diff
@@ -0,0 +1,48 @@
+## Packagers
+
+**Packagers** should take the auto-generated
+`org.hydrogenmusic.hydrogen.metainfo.xml` and
+`org.hydrogenmusic.hydrogen.releases.xml` files.
+
+The older `org.hydrogenmusic.Hydrogen.appdata.xml` file is kept for
+backward compatibility of existing packaging workflows (at least for
+now). But in general the newer and properly licensed
```

Comment:
there is no point keeping it around. There shall only be one.

Reply:
I had the idea to keep it around for one release and mark it deprecated. But I'm also fine to remove it once the newer version has stabilized.